CHANGELOG.md

```diff
@@ -1,3 +1,34 @@
+* **experimental**: Add client library API to list recently updated jobs ([`bd32564`](https://github.com/aws-deadline/deadline-cloud/commit/bd325640e573bfabc8865b8a91f6158a733c7e75))
+* **experimental**: command bootstrap logic and reading from, saving to state file for incremental downloads cli (#691) ([`7acfcba`](https://github.com/aws-deadline/deadline-cloud/commit/7acfcba3271e633c81c0887d1a049ebd933a97d0))
+* **experimental**: command bootstrap logic and reading from, saving to state file for incremental downloads cli ([`7acfcba`](https://github.com/aws-deadline/deadline-cloud/commit/7acfcba3271e633c81c0887d1a049ebd933a97d0))
+* **experimental**: command bootstrap logic and reading from, saving to state file for incremental downloads cli (#691) ([`b0488a6`](https://github.com/aws-deadline/deadline-cloud/commit/b0488a62bad8927b888dff0f0617a9cbc477d729))
+* **experimental**: command bootstrap logic and reading from, saving to state file for incremental downloads cli ([`b0488a6`](https://github.com/aws-deadline/deadline-cloud/commit/b0488a62bad8927b888dff0f0617a9cbc477d729))
+* Adding a helper to initialize the lru_cached get_s3_client method. Adding a cached get_session_client method to allow get_boto3_client calls which end up using the same session and same service name to use cached clients. (#708) ([`5665fa8`](https://github.com/aws-deadline/deadline-cloud/commit/5665fa8813ef8bb5c01a57c9604bbaed7015d859))
+* **experimental**: fix atomicity for pid lock & add release lock support for incremental downloads cli (#682) ([`da853ac`](https://github.com/aws-deadline/deadline-cloud/commit/da853ac0b5e1d15e7e2c60d908e71f81d0b39521))
+* **experimental**: pid locking atomicity fixes for automatic incremental downloads ([`da853ac`](https://github.com/aws-deadline/deadline-cloud/commit/da853ac0b5e1d15e7e2c60d908e71f81d0b39521))
+
+### Bug Fixes
+* installer_version.txt was not uninstalled (#736) ([`3f880b8`](https://github.com/aws-deadline/deadline-cloud/commit/3f880b8114bd65ffa5d6663ca7e0c99a5399ef79))
+* profile settings gui widget widths are truncated (#724) ([`5fdb0f1`](https://github.com/aws-deadline/deadline-cloud/commit/5fdb0f1ca347ba082bb48556aa57e9128d930e11))
+* normalize paths in windows so relative_to works properly (#715) ([`2e70d5d`](https://github.com/aws-deadline/deadline-cloud/commit/2e70d5d9b894630d8711369ca1509bf2f0a082fb))
+* Implement improved job attachments containment warning ([`a8bf21b`](https://github.com/aws-deadline/deadline-cloud/commit/a8bf21bc1af85b6f67694defdb93fee49f3516a6))
+* handle normalizing windows paths when they are larger than MAX_PATH ([`b841f61`](https://github.com/aws-deadline/deadline-cloud/commit/b841f617047cc6a6354a6f694b7bac2423e2dcee))
+* fix PATH env var deletion on uninstall ([`fcf4a75`](https://github.com/aws-deadline/deadline-cloud/commit/fcf4a75c01d733c37ee509a156589ff8911dc4cf))
```

Comment:
This will require a bit of PR diving, but can we modify the bug fixes here to describe what the bug would appear like to the end user? Example, the first one should be something like

`Uninstall would leave a file behind` or something similar